.github/workflows/integration-test.yaml

```diff
@@ -102,31 +104,60 @@ jobs:
-            --vnet-name ${{ env.db2VmName }}VNET --enable-auto-update false \
+            --vnet-name ${{ env.dbVmName }}VNET --enable-auto-update false \
             --tags SkipASMAzSecPack=true SkipNRMSCorp=true SkipNRMSDatabricks=true SkipNRMSDB=true SkipNRMSHigh=true SkipNRMSMedium=true SkipNRMSRDPSSH=true SkipNRMSSAW=true SkipNRMSMgmt=true
-          az vm open-port -g ${{ env.testResourceGroup }} -n ${{ env.db2VmName }} --port 50000 --priority 100
+          az vm open-port -g ${{ env.testResourceGroup }} -n ${{ env.dbVmName }} --port 50000 --priority 100
           az vm extension set --name CustomScript \
             --extension-instance-name install-db2-in-docker \
-            --resource-group ${{ env.testResourceGroup }} --vm-name ${{ env.db2VmName }} \
+            --resource-group ${{ env.testResourceGroup }} --vm-name ${{ env.dbVmName }} \
             --publisher Microsoft.Azure.Extensions --version 2.0 \
             --settings "{\"fileUris\": [\"${{ env.scriptLocation }}install-db2.sh\"]}" \
-            --protected-settings "{\"commandToExecute\":\"bash install-db2.sh ${{ env.db2inst1Password }}\"}"          
+            --protected-settings "{\"commandToExecute\":\"bash install-db2.sh ${{ env.db2inst1Password }}\"}"
+      - name: Deploy an Oracle database server on Azure VM
+        if: ${{ inputs.databaseType == 'oracle' || github.event.client_payload.databaseType == 'oracle' }}
+        run: |
+          az group create -n ${{ env.testResourceGroup }} -l ${{ env.location }}
+          az vm create \
+            --resource-group ${{ env.testResourceGroup }} --name ${{ env.dbVmName }} \
+            --image Oracle:oracle-database-19-3:oracle-database-19-0904:latest --size Standard_DS2_v2 \
+            --admin-username azureuser --generate-ssh-keys \
+            --nsg-rule NONE --enable-agent true \
+            --vnet-name ${{ env.dbVmName }}VNET --enable-auto-update false \
+            --tags SkipASMAzSecPack=true SkipNRMSCorp=true SkipNRMSDatabricks=true SkipNRMSDB=true SkipNRMSHigh=true SkipNRMSMedium=true SkipNRMSRDPSSH=true SkipNRMSSAW=true SkipNRMSMgmt=true
+          az vm disk attach --name oradata01 --new --resource-group ${{ env.testResourceGroup }} --vm-name ${{ env.dbVmName }} --size-gb 64 --sku StandardSSD_LRS
+          az vm open-port -g ${{ env.testResourceGroup }} -n ${{ env.dbVmName }} --port 1521,5502 --priority 100
+          az vm extension set --name CustomScript \
+            --extension-instance-name install-oracle \
+            --resource-group ${{ env.testResourceGroup }} --vm-name ${{ env.dbVmName }} \
+            --publisher Microsoft.Azure.Extensions --version 2.0 \
+            --settings "{\"fileUris\": [\"${{ env.scriptLocation }}install-oracle-main.sh\", \"${{ env.scriptLocation }}install-oracle.sh\"]}" \
+            --protected-settings "{\"commandToExecute\":\"bash install-oracle-main.sh ${{ env.oracleDBPassword }}\"}"           

```

Comment:
Amazing stuff here.

Reply:
I haven't looked into it as Galia shared me an [official doc](https://learn.microsoft.com/en-us/azure/virtual-machines/workloads/oracle/oracle-database-quick-create) from `learn.microsoft.com` which installs an Oracle server on an Azure VM. 
Thanks for sharing [oracle container registry database image](https://container-registry.oracle.com/ords/f?p=113:4:16393593035915:::4:P4_REPOSITORY,AI_REPOSITORY,AI_REPOSITORY_NAME,P4_REPOSITORY_NAME,P4_EULA_ID,P4_BUSINESS_AREA_ID:8,8,Oracle%20Database%20Standard%20Edition%202,Oracle%20Database%20Standard%20Edition%202,1,0&cs=3p0HpayI5XMUmDL4HWFy3lBevy7cDiY4UNmKz6wbqcfnCA4yncP5KlQZq1KLHHdgBfzJREF2VhTwXVGxtsRroQQ), it looks like another option to install Oracle server.